LICENSE

```diff
@@ -2,7 +2,8 @@ The following license applies to all source codes and assets, with the
 exception of Code.org's proprietary or licensed artwork, graphics, sounds,
 lectures, videos, logos, characters, fonts, etc. ("Proprietary Materials").
 
-The Proprietary Materials, a list of which [can be found here](https://github.com/code-dot-org/code-dot-org/blob/staging/NOTICE), may
+The Proprietary Materials, a list of which can be found here
+(https://github.com/code-dot-org/code-dot-org/blob/staging/NOTICE), may
```

Comment:
This is just updating the formatting so the text wraps with the rest of the paragraph